frontend/src/components/AirdropBanner/index.tsx

```diff
@@ -0,0 +1,75 @@
+import { useState } from 'react';
+import { motion, useMotionValue, AnimatePresence } from 'motion/react';
+
+export const AirdropBanner = () => {
+  const [expanded, setExpanded] = useState(false);
+  const rotation = useMotionValue(0);
+
+  const toggleExpand = () => {
+    rotation.set(expanded ? 0 : 180);
+    setExpanded(!expanded);
+  };
+
+  return (
+    <motion.div
+      className="bg-[#e8b923] text-black mt-12 mx-3 rounded-lg shadow-custom md:mx-auto md:w-96 px-4 py-2 cursor-pointer"
+      onClick={toggleExpand}
+      initial={{ opacity: 0, y: -20 }}
+      animate={{ opacity: 1, y: 0 }}
+      transition={{ duration: 0.5, ease: 'easeOut' }}
+      whileHover={{ scale: 1.01 }}
+    >
+      <div className='flex flex-col items-center justify-center'>
+       <div className="flex items-center justify-between text-center">
+        <div>
+       <h1 className='text-xl'>🪂<strong> Airdrop offer</strong></h1>
```

Comment:
Let's maybe not call it airdrop and stick to the 'special' wording. 
```suggestion
       <h1 className='text-xl'>🪂<strong> Special offer</strong></h1>
```